src/daemon/message_hexdump.go

```diff
@@ -0,0 +1,255 @@
+package daemon
+
+import (
+	"fmt"
+	"strconv"
+
+	"io/ioutil"
+	"os"
+
+	"github.com/skycoin/skycoin/src/cipher/encoder"
+	"github.com/skycoin/skycoin/src/daemon/gnet"
+
+	"reflect"
+
+	"github.com/skycoin/skycoin/src/util"
+)
+
+var registered = false
+
+func getSliceContentsString(sl []string, offset int) string {
+	var res = ""
+	var counter = 0
+	var currentOff = offset
+	if offset != -1 {
+		var hex = strconv.FormatInt(int64(offset), 16)
+		var l = len(hex)
+		for i := 0; i < 4-l; i++ {
+			hex = "0" + hex
+		}
+		hex = "0x" + hex
+		res += hex + " | "
+	}
+	for i := 0; i < len(sl); i++ {
+		counter++
+		res += sl[i] + " "
+		if counter == 16 {
+			res += "\n"
+			currentOff += 16
+			if offset != -1 {
+				//res += "         " //9 spaces
+				var hex = strconv.FormatInt(int64(currentOff), 16)
+				var l = len(hex)
+				for i := 0; i < 4-l; i++ {
+					hex = "0" + hex
+				}
+				hex = "0x" + hex
+				res += hex + " | "
+			}
+			counter = 0
+		}
+	}
+	for i := 0; i < (16 - counter); i++ {
+		res += "..."
+	}
+	res += "..."
+	return res
+}
+
+func printLHexDumpWithFormat(offset int, name string, buffer []byte) {
```

Comment:
@stdevEdu what is this function for ? Pls add one-line comment.

Reply:
This method was also a helper fot NonGenericHexDump, so it was removed too.